.github/CODEOWNERS

```diff
@@ -0,0 +1,36 @@
+# Code owners will be used to identify maintainers, notably
+# for developmental and contributed code
+
+# Overall (non-specific)
+* @lbianchi-lbl @ksbeattie
+
```

Comment:
```suggestion
# packaging
setup.py @lbianchi-lbl @ksbeattie
requirements*.txt @lbianchi-lbl @ksbeattie 

# CI infra
.github/ @lbianchi-lbl @ksbeattie 

# testing
pytest*.ini @lbianchi-lbl
conftest.py @lbianchi-lbl
test_*.py @lbianchi-lbl  # this might prove to be too much
```

Reply:
I'll have to double-check if the syntax is correct for single files

Reply:
@lbianchi-lbl I think the test one would need to go at the end - later lines override earlier lines so this would only assign you to tests that were not covered by something else.

Also, I was intending for the maintainers to be responsible for their own tests - I don't know if you really want to be tagged on everything testing related.

Reply:
Also, I think (hope) the first line will assign you and Keith to anything not covered by something else.